libcodechecker/cmd/cmd_line_client.py

```diff
@@ -647,10 +650,19 @@ def checker_count(checker_dict, key):
             intentional=checker_count(intentional_checkers, key),
             resolved=checker_count(resolved_checkers, key),
          ))
+        total['total_reports'] += checker_data.count
+        total['total_resolved'] += checker_count(resolved_checkers, key)
+        total['total_unreviewed'] += checker_count(unrev_checkers, key)
+        total['total_confirmed'] += checker_count(confirmed_checkers, key)
+        total['total_false_positive'] += checker_count(false_checkers, key)
+        total['total_intentional'] += checker_count(intentional_checkers, key)
 
     if args.output_format == 'json':
         print(CmdLineOutputEncoder().encode(all_results))
+        print(CmdLineOutputEncoder().encode(total))
         print(CmdLineOutputEncoder().encode(severities))
+        print(CmdLineOutputEncoder().encode([{
```

Comment:
I do not think we should print out the total and number of all reports here.
With the current setup we print out not just one json. This api is used for further scripts to process the results, they can count it if they need it.

I think this line should be removed too:
https://github.com/Ericsson/codechecker/blob/02b70fe244c9d2dec632e8a797203b5e44a96ac6/libcodechecker/cmd/cmd_line_client.py#L663